packages/build/tests/helpers/tcp_server.js

```diff
@@ -1,17 +1,29 @@
 const { createServer } = require('net')
 const { promisify } = require('util')
 
+const getPort = require('get-port')
 const { tmpName } = require('tmp-promise')
 
 // Start a TCP server to mock calls.
-const startTcpServer = async function({ response = '', timeout = false } = {}) {
+const startTcpServer = async function({ response = '', useUnixSocket = false, timeout = false } = {}) {
```

Comment:
[sand] Can we change the test default to reflect the buildbot default (`useUnixSocket = true`)?

Reply:
Fixed :+1: